tests/tensor/test_math.py

```diff
@@ -3557,3 +3557,15 @@ def test_deprecations():
 
     with pytest.deprecated_call():
         from aesara.tensor.math import true_div  # noqa: F401 F811
+
+    with pytest.deprecated_call():
+        from aesara.tensor import floor_div  # noqa: F401 F811
+
+    with pytest.deprecated_call():
+        from aesara.tensor.math import floor_div  # noqa: F401 F811
+
+    with pytest.deprecated_call():
+        from aesara.tensor import int_div  # noqa: F401 F811
+
+    with pytest.deprecated_call():
+        from aesara.tensor.math import int_div  # noqa: F401 F811
```

Comment:
I have a question here: the test for deprecation of `floor_div` is working fine, but the test for `int_div` is not working; could you please point out where I went wrong?

Reply:
Try to rebase your branch on `main`, I think this was fixed in https://github.com/aesara-devs/aesara/pull/1434

Reply:
I have already tried this, you can the tests which I have added for checking `floor_div` is working fine but the tests for `int_div` is not working, don't know what is the issue here, they both present at the same location.